content/ideas/json-autotype-other-languages.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Expand json-autotype support to other functional programming languages
+---
+
+[json-autotype](http://github.com/mgajda/json-autotype) is currently the *most advanced* type and parser generator from JSON to Haskell. Using union types it works as most accurate type providers.
+
+It was of general interest to enhance it to produce types and parsers for other strongly-typed programming languages like Elm, PureScript, Scala, Elixir, F#, or Java.
+Beside practical importance, and will allow us to pitch Haskell as leading language to other strongly-typed language communities..
+
+**Mentor**: [Michal Gajda](mailto:migamake&#64;migamake.com)
+
+**Difficulty**: Light
+
```

Comment:
We choose projects based on how the benefit the Haskell community.  If this comes down to effectively porting a Haskell library to another language, I don't think it makes sense to host this under Haskell.org's Google Summer of Code organization.  If someone wants to port it to e.g. Scala, then they should do that within Scala's organization.